packages/katello/katello/backup.rb

```diff
@@ -364,6 +375,11 @@ def setup_opt_parser
           end
         end
 
+        opts.on("--split-pulp-tar SIZE", "Split pulp data into files of a specified size, see 'tar --tape-length' for syntax") do |size|
+          opts.abort("Please specify size according to 'tar --tape-length' format.") unless size
```

Comment:
and is impossible to find the actual size command table! looks like `info tar` and some scrolling will show tape-length sizes. 

How about this:
"Split pulp data into files of a specified size, i.e. (100M, 50G). See '--tape-length' in 'info tar' for all sizes"
This should save most users from digging around tar docs

Also what happens if I pass `100XYZ` in there? This will get passed to the tar command and it will error. I think there should be a validation method here, let me know if you need help writing it.

Reply:
> let me know if you need help writing it.

That would be very helpful.  I don't actually know ruby and am just programing by pattern....